xbmc/filesystem/NFSDirectory.cpp

```diff
@@ -25,9 +25,11 @@
 #endif
 
 using namespace XFILE;
+#include <algorithm>
```

Comment:
This header is still necessary?

Also `#include "utils/StringUtils.h"` can be removed now

Unrelated, but previous Windows Jenkins failures was because missing <array> header (not necessary now with latest force push). Anyway seems better squash commits to avoid problems.